Makefile

```diff
@@ -188,6 +188,10 @@ format:
 
 fmt: format
 
+lint:
+	# tests/utils.go is too long already. Don't make it worse
```

Comment:
Maybe we can add a comment here about the fact that this is just a place-holder for a future bash script that will perform a bit more sophisticated check?